src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,77 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final Pair[] elements = new Pair[MAX_SIZE];
+    private int size = 0;
+
+    public StorageImpl() {
+    }
```

Comment:
```suggestion
    private final Pair[] elements;
    private int size;

    public StorageImpl() {
        elements = new Pair[MAX_SIZE];
    }
```

![image](https://github.com/mate-academy/jv-storage/assets/20326858/b3c9b059-bb5c-4fa6-8373-a66034aaa7ce)